### ISSUE TITLE
[FEAT]: 

### ISSUE THREAD

**Issue by Windows11-MEMZ** [BartoszCichecki/LenovoLegionToolkit#665]
### Version

2.11.2

### OS

WIndows1122H2 25536.1000

### Device

Legion R9000P ARH7H

### Is your feature request related to a problem?

in my laptop there's a big "Y" logo,can u add the function that can control it

### How would you like the problem to be solved?

add the function that can control it

### What alternatives have you considered?

no

**Comment by BartoszCichecki** [comment]
Duplicate of #342 and #649.

Shortly speaking, if someone knows a way go for it, but I couldn't find a reliable way to control it.

**Comment by BartoszCichecki** [comment]
> I think you kan use the way which can control keyboard light to control it

Unfortunately not - I checked that already. If someone comes up with an idea how to control it, I will definitely add it.

**Comment by Windows11-MEMZ** [comment]
just like this
<img width="702" alt="image" src="https://user-images.githubusercontent.com/107833760/231169857-76ca9a04-acc0-4ab3-9ca4-58e427e5cbc1.png">

**Comment by BartoszCichecki** [comment]
Yeah, I have that one. But it's incredibly hard to reverse engineer, so I couldn't figure it out. I could control the panel logo on Legion 5 Pro, but Legion 7 uses different one and what is more many laptops report that they have the logo but they don't. So the situation is incredibly messy.

**Comment by Windows11-MEMZ** [comment]
how can you reverse engineer,i can try it

**Comment by BartoszCichecki** [comment]
With this for example: https://github.com/NationalSecurityAgency/ghidra

**Comment by Windows11-MEMZ** [comment]
> With this for example: https://github.com/NationalSecurityAgency/ghidra

emm…… can you change it to drive?
it is so slow

**Comment by Windows11-MEMZ** [comment]
[odbg201.zip](https://github.com/BartoszCichecki/LenovoLegionToolkit/files/11200869/odbg201.zip)


> With this for example: https://github.com/NationalSecurityAgency/ghidra

you can use it

**Comment by BartoszCichecki** [comment]
Look, I already tried. If you or someone else wants to implement it, pull requests are welcome, but I will not spend more time on this.

**Comment by Windows11-MEMZ** [comment]
> 

ok thank you

**Comment by Windows11-MEMZ** [comment]
ok i will go sleep(in china it's 21:38

**Comment by KarlLee830** [comment]
OK, honestly, these links you sent are Chinese articles and BC (aka the author) doesn't understand these Chinese articles.
Also these reverse tools are widely known and searching for these Chinese articles on search engines and sending them here is not very helpful.MAN You should really go through the basic etiquette of creating a new issue on Github. 
Try not to let the free open source authors waste their time answering these somewhat noobish questions.
